postgraphile/src/config/parse.ts

```diff
@@ -37,6 +38,9 @@ export function parseConfig(
     gqSchemas = parsedToml['database']['gq_schemas'];
     gqUser = parsedToml['database']['gq_user'] || gqUser;
     gqPassword = parsedToml['database']['gq_password'] || gqPassword;
+    disableDefaultMutations = parsedToml['database']['disable_default_mutations'] === undefined
```

Comment:
I'm no TS/JS wizard, what's the difference between this and the disjunctions above?  I _think_ this would behave unexpectedly on an empty string for example.

Reply:
Do you mean if there were an empty string in the config.toml?  In that case, it'll ignore the incorrect var and keep the default `true`. I did it this way because there are expected cases where `disableDefaultMutations` will be falsey. 

So, if `disable_default_mutations = false` in the config.toml, `disableDefaultMutations = parsedToml['database']['disable_default_mutations'] || true;` would still end up true.

Reply:
Ah, makes sense :ok_hand: